src/vs/workbench/contrib/terminal/common/terminalEnvironment.ts

```diff
@@ -119,7 +119,7 @@ function _getLangEnvVariable(locale?: string) {
 	return parts.join('_') + '.UTF-8';
 }
 
-export function getCwd(shell: IShellLaunchConfig, userHome: string, root?: Uri, customCwd?: string): string {
+export function getCwd(shell: IShellLaunchConfig, userHome: string, lastActiveWorkspace: IWorkspaceFolder | undefined, configurationResolverService: IConfigurationResolverService | undefined, root?: Uri, customCwd?: string): string {
```

Comment:
Let's also chang eroot and customCwd to `| undefined` instead of `?` because they're always passed in